src/StateSmithTest/Output/Sim/diagrams/BeadSorter.sim.html

```diff
@@ -3159,8 +3159,21 @@
         // choose to implement a tracer for debugging purposes.
         sm.tracer = {
             enterState: (mermaidName) => {
-                document.querySelector('g[data-id=' + mermaidName + ']')?.classList.add('active');
-                sm.tracer.log("➡️ Entered " + mermaidName);
+                var e = document.querySelector('g[data-id=' + mermaidName + ']');
+                e?.classList.add('active');
```

Comment:
instead of propagating `?.` in a number of places, could we just do an early return if `e` is null/undefined at the top? I would find the code a bit easier to follow as there's less to consider.